### PR TITLE
Update header on Send page to be more accurate

### LINKED ISSUE #11875: Change header text in MetaMask send flow
The header on the below screen should say "Send to" instead of "Add recipient"

Screenshot_2021-07-13_at_17 39 06

### PR DESCRIPTION
**Pull request by mcmire**
Change "Add Recipient" to "Send to".

Fixes: #11875.

Manual testing steps:  
* Go to home
* Click on "Send"
* Header should read "Send to"
* Click on avatar
* Change language to something else, e.g., Spanish
* Click on equivalent of "Send"
* Header should still read "Send to"

---

Screen Shot 2021-08-19 at 12 10 48 PM

### CODE REVIEW

File: ui/pages/send/send-header/send-header.component.test.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ describe('SendHeader Component', () => {
           history: { mostRecentOverviewPage: 'activity' },
         }),
       );
-      expect(getByText('Add Recipient')).toBeTruthy();
+      expect(getByText('Send to')).toBeTruthy();
     });
 
     it('should render "Send" for DRAFT stage when asset type is NATIVE', () => {
```
Comment (mcmire) [inline ui/pages/send/send-header/send-header.component.test.js:45]: Do we want to keep this existing behavior or change this as well?

File: ui/pages/send/send.test.js [hunk 2]
```diff
@@ -146,7 +146,7 @@ describe('Send Page', () => {
     });
   });
 
-  describe('Send and Edit Flow', () => {
+  describe('Send and Edit Flow (draft)', () => {
```
Comment (mcmire) [inline ui/pages/send/send.test.js:149]: I am not sure what the "draft" stage is or how to reproduce it by hand. I tacked "(draft)" onto the end of this `describe` to differentiate it from the new wording on line 127 — any thoughts on a better way to word this or is this clear as-is?

Comment (danjm) [reply ui/pages/send/send.test.js:149]: this works!

File: ui/pages/send/send-header/send-header.component.test.js (hunk 1 above)
Comment (danjm) [reply ui/pages/send/send-header/send-header.component.test.js:45]: This is fine to keep

**Review by danjm — APPROVED** [review]

**Review by ryanml — APPROVED** [review]
Great work!

**Pull request merged**